### PR TITLE
mrc-2999 add run report app with tabs under new vuex route

### PR DESCRIPTION
**Pull request by hillalex**
Adds the parent Vue+Vuex app for the run report page, with a top level tabs component and skeleton components for each tab. ~~Builds off https://github.com/vimc/orderly-web/pull/445 so that should be merged first and the base here pointed back to master.~~

### CODE REVIEW

File: src/app/static/src/js/components/documents/fileIcon.vue [hunk 1]
```diff
@@ -7,6 +7,6 @@
 
 <script>
     export default {
-        name: "fileIcon"
```
Comment (hillalex) [inline src/app/static/src/js/components/documents/fileIcon.vue:10]: Unrelated to changeset, guess this is cos I ran `npm run lint -- --fix` and it fixed this warning.

**Review by LekanAnni — CHANGES_REQUESTED** [review]
Looks great, 

Some missing tests such as mutation test.

File: src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/vuex/RunReportPageTests.kt [hunk 2]
```diff
@@ -35,9 +35,9 @@ class RunReportPageTests
     }
 
     @Test
-    fun `renders run report header correctly`()
+    fun `renders run report tabs correctly`()
     {
-        val runReportComponent = doc.select("#vuexRunReportApp").select("h1")
-        assertThat(runReportComponent.text()).isEqualTo("Vuex Run Report Page")
+        doc.select("#runReportTabsVueApp").select("run-report-tabs")
```
Comment (EmmaLRussell) [inline src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/vuex/RunReportPageTests.kt:40]: I don't think the select will cause the test to fail if no element is found, you'll just get an empty list.
```suggestion
        val tabs = doc.select("#runReportTabsVueApp").select("run-report-tabs")
        assertThat(tabs.count()).isEqualTo(1)
```

**Review by EmmaLRussell — COMMENTED** [review]
LGTM, just one query on the template test

**Review by LekanAnni — APPROVED** [review]
LGTM 👍

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**